bigflow_python/python/bigflow/serde.py

```diff
@@ -637,7 +637,7 @@ def of(rtype):
         return rtype
 
     if isinstance(rtype, entity.EntitiedBySelf):
-        raise "Not Support this serde :", rtype
+        raise Exception("Not Support this serde :", rtype)
```

Comment:
cc @acmol. 

Generally speaking, we shouldn't raise the generic Exception directly. But, since we already used Exception in serde.py, I am ok with this change. We can replace these Exceptions in following PRs.